packages/legacy-client/src/client/interfaces/aggregations.ts

```diff
@@ -111,15 +112,17 @@ export type GroupByFromType<T, N extends string> = NonNullable<T> extends number
   : T extends Array<infer U> ? GroupByFromType<U, N>
   : never;
 
-export type AggregationFromType<T> = NonNullable<T> extends number
+export type AggregationFromType<T> = NonNullable<T> extends StringLong
```

Comment:
This is the main change that adds numerical aggregation support back to long-type properties